Download all .fastq files in a project/Download all .fastq files in a project.ipynb

```diff
@@ -0,0 +1,6347 @@
+{
```

Comment:
<p>This needs a bit more clarity so people have some kind of rule of thumb to decide when to use <code>client.post_search</code> versus <code>client.post_search.iterate</code>. (It seems like having a lot of bundles is the way you make a decision, but from the long list above, it _already_ looks like there are a lot of bundles, so it isn't clear how many bundles "a lot of bundles" is. Also "a long time" is ambiguous - is "long" a few seconds, or a few hours?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCellAtlas/data-consumer-vignettes/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>